src/createBaseForm.js

```diff
@@ -452,7 +452,7 @@ function createBaseForm(option = {}, mixins = []) {
           const name = field.name;
           if (options.force !== true && field.dirty === false) {
             if (field.errors) {
-              set(alreadyErrors, name, { errors: field.errors });
+              alreadyErrors[name] = { errors: field.errors };
```

Comment:
but `alreadyErrors` will still be an object, the callback looks like just uses the attribute name of the `alreadyErrors` to get the corresponding error message.

https://github.com/react-component/form/blob/master/src/createDOMForm.js#L63

The other solution maybe change the way `validateFieldsAndScroll` to recursive look for error message that would be more complicated.

Haven't found another use of the callback

we really need this feature or our forms would not work consistent :(